lib/Mac/PropertyList.pm

```diff
@@ -953,6 +967,20 @@ use base qw(Mac::PropertyList::Scalar);
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 package Mac::PropertyList::data;
 use base qw(Mac::PropertyList::Scalar);
+use HTML::Entities;
```

Comment:
We don't need this here because the character set used by Base64 doesn't include any of the characters that we would want to encode.